.travis.yml

```diff
@@ -177,6 +177,18 @@ jobs:
 #          distributions: sdist bdist_wheel
 #          repo: raiden-network/raiden
 #          branch: master
+      - provider: s3
+        access_key_id: MK22XIZDUYA3EWBHGAKD
```

Comment:
This also needs `skip_cleanup: true`, otherwise travis will perform a `git stash -a` (which removes all not git-tracked files from the working tree) and the build will be gone.

Reply:
Added.